### PR TITLE
Moved extensions from kafka to common

### PR DESCRIPTION
**Pull request by eugene-cheverda**

### CODE REVIEW

File: project/local.scala [hunk 1]
```diff
@@ -8,8 +8,16 @@ object local {
...
+      }
     }
 
+    object finagle {
+      val core   = "com.twitter" %% "finagle-http"   % versions.twitter.finagle
+    }
+
+
     object log4s {
       val core = "org.log4s" %% "log4s" % versions.log4s
```
Comment (theocat) [inline project/local.scala]: @eugene-cheverda let's move this to `sbt-pulse`

File: project/local.scala [hunk 2]
```diff
@@ -8,8 +8,16 @@ object local {
 
     object versions {
       val log4s = "1.3.4"
+      object twitter {
+        val finagle = "6.42.0"
```
Comment (theocat) [inline project/local.scala]: @eugene-cheverda let's move this to `sbt-pulse`

Comment (eugene-cheverda) [inline project/local.scala]: done

File: project/local.scala (hunk 1 above)
Comment (eugene-cheverda) [inline project/local.scala]: done

**Pull request merged**